sdk/workflow_run.go

```diff
@@ -77,10 +81,29 @@ type WorkflowNodeRunRelease struct {
 
 // WorkflowRunPostHandlerOption contains the body content for launch a workflow
 type WorkflowRunPostHandlerOption struct {
-	Hook        *WorkflowNodeRunHookEvent `json:"hook,omitempty"`
-	Manual      *WorkflowNodeRunManual    `json:"manual,omitempty"`
-	Number      *int64                    `json:"number,omitempty"`
-	FromNodeIDs []int64                   `json:"from_nodes,omitempty"`
+	Hook         *WorkflowNodeRunHookEvent `json:"hook,omitempty"`
+	Manual       *WorkflowNodeRunManual    `json:"manual,omitempty"`
+	Number       *int64                    `json:"number,omitempty"`
+	FromNodeIDs  []int64                   `json:"from_nodes,omitempty"`
+	AuthConsumer *AuthConsumer             `json:"auth_consumer,omitempty"`
```

Comment:
Looks like its used for event, can we store only an identifiable ?

Reply:
done